pkg/ddevapp/ddevapp.go

```diff
@@ -388,6 +390,22 @@ func (app DdevApp) GetDocroot() string {
 	return app.Docroot
 }
 
+// GetAppRoot return the full path from root to the app directory
+func (app *DdevApp) GetRelativeWorkingDirectory() (string, error) {
+	re, err := regexp.Compile(app.AppRoot + `/`)
+
+	if err != nil {
+		return ``, err
+	}
+
+	pwd, err := os.Getwd()
+	if err != nil {
+		return ``, err
+	}
+
+	return re.ReplaceAllString(pwd, ""), nil
```

Comment:
You don't want more than one replacement do you? A simple string replacement would work wouldn't it? strings.TrimPrefix()?